### ISSUE TITLE
Why is there no training process? Loss is not the same as paper.

### ISSUE THREAD

**Issue by jiangzhiping** [wangruixing/DeepUPE#16]
Why is there no training process? The loss is not described in the same way as the article.

**Comment by wangruixing** [comment]
What is the difference? The color loss is writing in training code. Did you mean 'lamda' parameter? It is just a weight parameter.
